src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements Draw, AreaCounter {
+    private String color;
```

Comment:
Can u use the enum here? It is better to avoid primitives if it is possible

Reply:
Yes,it`s done!)